packages/web/app/views/programRegistry/ConditionSection.jsx

```diff
@@ -39,9 +40,52 @@ const ClippedConditionName = styled.span`
   -webkit-line-clamp: 2;
   -webkit-box-orient: vertical;
   word-wrap: break-word;
-  width: 95%;
+  width: 100%;
+`;
+
+const ConditionCategory = styled.span`
+  color: ${Colors.midText};
 `;
 
+const getGroupedConditions = (conditions) => {
+  const openConditions = [];
+  const closedConditions = [];
+
+  conditions.forEach(condition => {
+    if (condition.conditionCategory === PROGRAM_REGISTRY_CONDITION_CATEGORIES.recordedInError) {
+      return;
+    }
+
+    if ([
+      PROGRAM_REGISTRY_CONDITION_CATEGORIES.resolved,
+      PROGRAM_REGISTRY_CONDITION_CATEGORIES.disproven,
+    ].includes(condition.conditionCategory)) {
+      closedConditions.push(condition);
+      return;
+    }
+
+    openConditions.push(condition);
+  });
+
+  return { openConditions, closedConditions };
+};
+
+// TODO: Translate program registry condition name and category, plus sort by that new display name
+const ConditionMapperFn = (condition) => {
+  const { programRegistryCondition, conditionCategory } = condition;
+  const { name } = programRegistryCondition;
+  const fullText = `${name} (${conditionCategory})`;
+  return (
+    <ConditionalTooltip title={fullText} visible={fullText.length > 65} key={condition.id}>
+      <Condition>
+        <ClippedConditionName>
+          {name} <ConditionCategory>({conditionCategory})</ConditionCategory>
```

Comment:
```suggestion
<TranslatedEnum value={conditionCategory} enumValues={PROGRAM_REGISTRY_CONDITION_CATEGORIES} />
```
As we discussed, we could use the TranslatedEnum here

Reply:
Okay so this doesn't really work because we're saving the "labels" as values on the database, so the keys do not match the enums 🤔 I think the proper way to go is to follow the same pattern elsewhere, have a constant with the values we'll save in the database, then have a constant that maps labels

Reply:
Or maybe these were just saved before the change 🤔 I think it might be that by looking at the code hey

Reply:
Yep, seems to be that. Ran these queries to cleanup the epic branch:

```
app=# UPDATE patient_program_registration_conditions SET condition_category = 'underInvestigation' WHERE condition_category = 'Under investigation';
UPDATE 2
app=# UPDATE patient_program_registration_conditions SET condition_category = 'disproven' WHERE condition_category = 'Disproven';
UPDATE 1
app=# UPDATE patient_program_registration_conditions SET condition_category = 'unknown' WHERE condition_category = 'Unknown';
UPDATE 3
app=# UPDATE patient_program_registration_conditions SET condition_category = 'confirmed' WHERE condition_category = 'Confirmed';
UPDATE 1
```

Reply:
Thanks 🙏